File: services/utils/date-utils.js
export function transformDateFormat(dateString) {
    // Parse the input date string
    const parts = dateString.split('/');
    const month = parts[0].padStart(2, '0');
    const day = parts[1].padStart(2, '0');
    const year = parts[2];

    // Construct the new date string in yyyy-mm-dd format
    return `${year}-${month}-${day}`;
}

File: services/utils/date_utils.py
def transform_date_format(date_string):
    # Parse the input date string
    parts = date_string.split('/')
    month = parts[0].zfill(2)
    day = parts[1].zfill(2)
    year = parts[2]

    # Construct the new date string in yyyy-mm-dd format
    return f"{year}-{month}-{day}"